src/ruby/meetings/app/models/source_file.rb

```diff
@@ -0,0 +1,8 @@
+class SourceFile < ApplicationRecord
```

Comment:
As mentioned with the last file, do we want to be consistent with putting spaces after commas?